### DISCUSSION TITLE
Only images were moved to path

### DISCUSSION THREAD

**Discussion by painky** [trganda/obsidian-attachment-management#98] [Q&A]
Hallo and thank you for great plugin. Iam new in obsidian and migrated all my notes from evernote to obsidian. The importer plugin moved all attachments to subfolder _resources/notename.resources/. I like that also with my new notes all attachments are moved to this folders. With you plugin its working fine for images but not for other attachments like .docx, xlsx, pdfs, etc.

Exclude patterns shoul not enabled. So iam not sure what happens.

My settings are
```
{
  "attachPath": {
    "attachmentRoot": "",
    "saveAttE": "obsFolder",
    "attachmentPath": "${notename}.resources",
    "attachFormat": "IMG-${date}",
    "type": "GLOBAL",
    "extensionOverride": []
  },
  "dateFormat": "YYYYMMDDHHmmssSSS",
  "excludeExtensionPattern": "",
  "autoRenameAttachment": true,
  "excludedPaths": "",
  "excludePathsArray": [],
  "excludeSubpaths": false,
  "originalNameStorage": [],
  "overridePath": {},
  "disableNotification": false
}
```

![image](https://github.com/trganda/obsidian-attachment-management/assets/40030075/bd3c4ac9-c3b3-4420-9db5-f1173755ba96)

Thank you in advance

**Comment by trganda** [comment]
Same as https://github.com/trganda/obsidian-attachment-management/issues/93 . You can use **extension override** to config attachemnt with other extension

<img width="833" alt="image" src="https://github.com/trganda/obsidian-attachment-management/assets/62204882/ae0adead-b60c-4849-bd35-aff8eeabc326">

**Reply by White-k7** [reply]
I followed your steps for **extension override** for an mp4 file, but it doesn't work.
How can we fix this?

**Reply by trganda** [reply]
Can you provide the verison of Obsidian and your configuration file of this plugin?
